python/TestHarness/schedulers/RunParallel.py

```diff
@@ -0,0 +1,130 @@
+from timeit import default_timer as clock
+
+from Tester import Tester
+from Scheduler import Scheduler
+from MooseObject import MooseObject
+from util import *
+import platform, time
+
+import os
+
+## This class provides an interface to run commands in parallel
+#
+# To use this class, call the .run() method with the tester options
+class RunParallel(Scheduler):
+    @staticmethod
+    def validParams():
+        params = Scheduler.validParams()
+        params.addRequiredParam('scheduler',    'RunParallel', "The name of this scheduler")
```

Comment:
I need it. But if you are sure, I need your help to solve the error I get when I remove it:
```python
Traceback (most recent call last):
  File "./run_tests", line 26, in <module>
    TestHarness.buildAndRun(sys.argv, app_name, MOOSE_DIR)
  File "/ssd/milljm/moose/python/TestHarness/TestHarness.py", line 30, in buildAndRun
    harness = TestHarness(argv, app_name, moose_dir)
  File "/ssd/milljm/moose/python/TestHarness/TestHarness.py", line 137, in __init__
    self.initialize(argv, app_name)
  File "/ssd/milljm/moose/python/TestHarness/TestHarness.py", line 498, in initialize
    self.scheduler = self.factory.create(plugin_params['scheduler'], self, plugin_params)
  File "/ssd/milljm/moose/python/FactorySystem/InputParameters.py", line 49, in __getitem__
    return self.valid[key]
KeyError: 'scheduler'
```